xml/operations-opsconsole-en-alarm_definitions.xml

```diff
@@ -90,28 +40,28 @@
   <orderedlist>
    <listitem>
     <para>
-     Click the <guimenu>Create Alarm Definitions</guimenu> button to open a
-     separate window that allows you to configure a new alarm.
+     Press <guimenu>Create Alarm Definition</guimenu> to open the
```

Comment:
GUI elements can only be clicked, hardware keys can only be pressed. Please use "Click \<guimenu\>Create Alarm\</guimenu\> to..."